src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -14,6 +14,7 @@ import AddTaskButton from './AddTaskButton';
 import API from '../../api';
 import messages from '../common/messages';
 import SidebarContent from './SidebarContent';
+import { AnnotatorState, withAnnotatorContext } from '../common/annotator-context';
```

Comment:
Are these changes still needed here?

Reply:
I could wrap the activity card with the context directly, but as of right now I'm passing it here to the activity feed.